src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,297 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "Ferruccio";
+    private static final String DEFAULT_PASSWORD = "df89(exZ45";
+    private static final int DEFAULT_AGE = 18;
+    private User user;
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
```

Comment:
storageDao is redundant actually. As described in common mistakes, let's use Storage.people directly